specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/examples/compute/CreateOrUpdateAVmWithCapacityReservation.json

```diff
@@ -0,0 +1,189 @@
+      "plan": {
+        "publisher": "microsoft-ads",
+        "product": "windows-data-science-vm",
+        "name": "windows2016"
+      },
+      "properties": {
+        "hardwareProfile": {
+          "vmSize": "Standard_DS1_v2"
+        },
+        "storageProfile": {
+          "imageReference": {
+            "sku": "windows2016",
+            "publisher": "microsoft-ads",
+            "version": "latest",
+            "offer": "windows-data-science-vm"
+          },
+          "osDisk": {
+            "caching": "ReadOnly",
+            "managedDisk": {
+              "storageAccountType": "Standard_LRS"
+            },
+            "createOption": "FromImage",
+            "name": "myVMosdisk"
+          }
+        },
+        "capacityReservation": {
+          "capacityReservationGroup": {
+            "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/CapacityReservationGroups/{crgName}"
+          }
+        },
+        "osProfile": {
+          "adminUsername": "{your-username}",
+          "computerName": "myVM",
+          "adminPassword": "{your-password}"
+        },
+        "networkProfile": {
+          "networkInterfaces": [
+            {
+              "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Network/networkInterfaces/{existing-nic-name}",
+              "properties": {
+                "primary": true
+              }
+            }
+          ]
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "myVM",
+        "properties": {
+          "osProfile": {
+            "adminUsername": "{your-username}",
+            "secrets": [],
+            "computerName": "myVM",
+            "windowsConfiguration": {
+              "provisionVMAgent": true,
+              "enableAutomaticUpdates": true
+            }
+          },
+          "capacityReservation": {
+            "capacityReservationGroup": {
+              "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/CapacityReservationGroups/{crgName}"
+            }
+          },
+          "networkProfile": {
+            "networkInterfaces": [
+              {
+                "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Network/networkInterfaces/nsgExistingNic",
+                "properties": {
+                  "primary": true
+                }
+              }
+            ]
+          },
+          "storageProfile": {
+            "imageReference": {
+              "sku": "standard-data-science-vm",
+              "publisher": "microsoft-ads",
+              "version": "latest",
+              "offer": "standard-data-science-vm"
+            },
+            "osDisk": {
+              "osType": "Windows",
+              "caching": "ReadOnly",
+              "diffDiskSettings": {
+                "option": "Local"
+              },
```

Comment:
remove diffDiskSettings as it is not mentioned in the request and is not a main line scenario.

Reply:
I still see diffDiskSettings in the response.